crates/parser/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "parser"
+version.workspace = true
+edition.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+datafusion.workspace = true
+datafusion_ext = { path = "../datafusion_ext" }
+protogen = { path = "../protogen" }
+prql-compiler = "0.11.3"
```

Comment:
Are there other crates that depend on prql? Can we make it just this crate that depends on it?

Reply:
yeah good catch, I forgot to remove that from `sqlexec` should be good to go now.